recipes/c-ares/all/conandata.yml

```diff
@@ -1,29 +1,47 @@
 sources:
+  "1.19.0":
+    url: "https://github.com/c-ares/c-ares/releases/download/cares-1_19_0/c-ares-1.19.0.tar.gz"
+    sha256: "bfceba37e23fd531293829002cac0401ef49a6dc55923f7f92236585b7ad1dd3"
   "1.18.1":
     url: "https://github.com/c-ares/c-ares/releases/download/cares-1_18_1/c-ares-1.18.1.tar.gz"
     sha256: "1a7d52a8a84a9fbffb1be9133c0f6e17217d91ea5a6fa61f6b4729cda78ebbcf"
   "1.17.2":
-    sha256: 444b7dbf33cb6e8a764d069e3485cbaaa28386d60eace6988275c3cb9534d300
-    url: https://github.com/c-ares/c-ares/archive/refs/tags/cares-1_17_2.tar.gz
+    url: "https://github.com/c-ares/c-ares/archive/refs/tags/cares-1_17_2.tar.gz"
+    sha256: "444b7dbf33cb6e8a764d069e3485cbaaa28386d60eace6988275c3cb9534d300"
   "1.17.1":
-    sha256: 61f7cf09605f5e38d4828f82d0e2ddb9de8e355ecfd6819b740691c644583b8f
-    url: https://github.com/c-ares/c-ares/archive/cares-1_17_1.tar.gz
+    url: "https://github.com/c-ares/c-ares/archive/cares-1_17_1.tar.gz"
+    sha256: "61f7cf09605f5e38d4828f82d0e2ddb9de8e355ecfd6819b740691c644583b8f"
   "1.16.1":
-    sha256: 870962cc8f6b352303c404ce848e2ea1f1072f3c0a940042209a72179511c08c
-    url: https://github.com/c-ares/c-ares/archive/cares-1_16_1.tar.gz
+    url: "https://github.com/c-ares/c-ares/archive/cares-1_16_1.tar.gz"
+    sha256: "870962cc8f6b352303c404ce848e2ea1f1072f3c0a940042209a72179511c08c"
   "1.15.0":
-    sha256: 7deb7872cbd876c29036d5f37e30c4cbc3cc068d59d8b749ef85bb0736649f04
-    url: https://github.com/c-ares/c-ares/archive/cares-1_15_0.tar.gz
+    url: "https://github.com/c-ares/c-ares/archive/cares-1_15_0.tar.gz"
+    sha256: "7deb7872cbd876c29036d5f37e30c4cbc3cc068d59d8b749ef85bb0736649f04"
   "1.14.0":
-    sha256: 62dd12f0557918f89ad6f5b759f0bf4727174ae9979499f5452c02be38d9d3e8
-    url: https://github.com/c-ares/c-ares/archive/cares-1_14_0.tar.gz
+    url: "https://github.com/c-ares/c-ares/archive/cares-1_14_0.tar.gz"
+    sha256: "62dd12f0557918f89ad6f5b759f0bf4727174ae9979499f5452c02be38d9d3e8"
```

Comment:
Would it be possible to take advantage of this modification to point to official tarballs (as already done for 1.18.1 & 1.19.0)?

Reply:
@SpaceIm 
Thanks a lot!
I modified urls.

Reply:
thanks, just one url to fix: https://github.com/conan-io/conan-center-index/pull/15523#discussion_r1092415978